### ISSUE TITLE
Can I use this library on Java? 

### ISSUE THREAD

**Issue by fastener** [starkware-libs/crypto-cpp#3]
Is this library used to generate stark signatures?

**Comment by andre77** [comment]
@fastener also looking for a library to generate STARK signatures
did you find any way around it ?

**Comment by Solpatium** [comment]
You can check https://github.com/software-mansion/starknet-jvm :)

**Comment by fercp** [comment]
@andre77  @fastener  check this one for pure java implementation https://github.com/fercp/starkex

**Comment by Cevedale** [comment]
Yes, you can use the StarkEx library with Java! StarkEx is a protocol for scalable and secure off-chain computation. While the library might be initially designed with a specific language in mind, you can often use it with other languages, including Java, through various methods.

One common approach is to use a library or tool that facilitates interoperability between languages, such as Java's native interface (JNI) for calling functions written in other languages like C or C++. If the StarkEx library provides a C or C++ interface, you can create a Java wrapper using JNI to make the functionality accessible from Java.

Keep in mind that you may need to check the documentation of the StarkEx library for any specific recommendations or available interfaces for Java integration. Additionally, consider looking for community-supported wrappers or projects that provide Java bindings for the StarkEx library
